content/docs/forms.md

```diff
@@ -64,29 +71,33 @@ class NameForm extends React.Component {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/VmmPgp?editors=0010)
+[**Probeer Het Op CodePen**](https://codepen.io/gaearon/pen/VmmPgp?editors=0010)
 
-Since the `value` attribute is set on our form element, the displayed value will always be `this.state.value`, making the React state the source of truth. Since `handleChange` runs on every keystroke to update the React state, the displayed value will update as the user types.
+Aangezien het `value` attribuut gezet is op ons form element, zal de weergegeven waarde altijd `this.state.value` zijn, waarbij de React state de enkele bron van waarheid is.
+Aangezien `handleChange` loopt bij elke toetsaanslag om de React state bij te werken, zal de weergegeven waarde bijgewerkt worden wanneer de gebruiker typt.
 
-With a controlled component, every state mutation will have an associated handler function. This makes it straightforward to modify or validate user input. For example, if we wanted to enforce that names are written with all uppercase letters, we could write `handleChange` as:
+Met een gecontroleerd component zal elke verandering van state een geassocieerde handler functie hebben.
+Dit maakt het heel eenvoudig om gebruikersinput te veranderen of te valideren.
+Bijvoorbeeld, als we willen afdwingen dat namen geschreven worden met alleen hoofdletters, kunnen we `handleChange` als volgt schrijven:
```

Comment:
```suggestion
Bijvoorbeeld, als we zouden willen afdwingen dat namen geschreven worden met alleen hoofdletters, zouden we `handleChange` als volgt kunnen schrijven:
```

It says `if we wanted ... we could write` and not `if we want to ... we can write`